dindctl

```diff
@@ -178,7 +209,7 @@ function cleanDind {
 }
 
 function logs {
-	docker exec -ti balena-container-$CONTAINER_NAME journalctl $@
+	docker exec -ti balena-container-$CONTAINER_NAME journalctl -f $@
```

Comment:
This'll of course follow by default, which is fine, just different from https://github.com/balena-io/balena-supervisor/pull/1307/files#diff-7e4e25b0630cffe29fc584f7a52c17bcR19